scripts/backup_etcd_disk_tenant_cluster.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+installation=$1
+cluster=$2
+
+if [ "$installation" == "" ] || [ "$cluster" == "" ]
+then
+    echo "Usage $0 <installation name> <cluster name>"
+    exit 1
+fi
+
+read -p "Installation: $installation, cluster: $cluster. Hit enter to continue, or ctrl+c to abort. "
+
+echo -n "Creating kubeconfig for $installation: "
+opsctl create kubeconfig -i $installation >/dev/null
+echo "OK"
+echo -n "Creating kubeconfig for $cluster @ $installation: "
+gsctl create kubeconfig --cluster=$cluster --certificate-organizations system:masters -e $installation >/dev/null
+echo "OK"
+
+echo -n "Getting master node name: "
+master="`kubectl --context=giantswarm-$cluster get no -l role=master|grep -v NAME|awk '{print $1}'`"
+
+if [ "$master" == "" ]
+then
+    echo "ERR"
```

Comment:
```suggestion
    echo "ERR: Master not found"
```

Reply:
this part of the script changed a lot already, it is more verbose now